ts/features/bonus/siciliaVola/screens/voucherList/VoucherDetailsScreen.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from "react";
```

Comment:
I got stress this screen with some unexpected data. It is pretty rely, I guess we have to ensure only some margins on labels 
<img src="https://user-images.githubusercontent.com/822471/135125870-f2bdc965-139e-42fb-8265-1ed2dcbddd88.png" height="700"/>

Reply:
Added space between labels and value and guaranteed that the text doesn't overflow the screen in [84411df](https://github.com/pagopa/io-app/pull/3400/commits/84411df39c8e21a09901d31634a176a1ac59f50e)